app/controllers/shared/api.php

```diff
@@ -238,6 +265,28 @@
     }
 
     $route = $utopia->match($request);
+
+    $useCache = $route->getLabel('cache', false);
+    if ($useCache) {
+            $key = md5($request->getURI() . $request->getServer('query_string'));
```

Comment:
getServer is not the optimal way to access this data, we might even want to deprecate it this method, as it is more useful with our legacy FPM server. `getParams()` should be better